lib/project_types/rails/commands/create.rb

```diff
@@ -70,7 +75,10 @@ def self.help
             {{command:--name=NAME}} App name. Any string.
             {{command:--app_url=APPURL}} App URL. Must be valid URL.
             {{command:--organization_id=ID}} App Org ID. Must be existing org ID.
-            {{command:--shop_domain=MYSHOPIFYDOMAIN }} Test store URL. Must be existing test store.
+            {{command:--shop_domain=MYSHOPIFYDOMAIN}} Test store URL. Must be existing test store.
+            {{command:--db=DB}} Database. Must be one of: mysql, postgresql, sqlite3, oracle, frontbase, ibm_db, sqlserver, jdbcmysql, jdbcsqlite3, jdbcpostgresql, jdbc.
+            {{command:--api}} For API only scaffolding.
+            {{command:--rails_opts=RAILSOPTS}} Additional flags. Must be string containing valid Rails flag(s), separated by space.
```

Comment:
Generally we refer to `options` rather than `flags`, so I'd make a slight change here:
```suggestion
            {{command:--rails_opts=RAILSOPTS}} Additional options. Must be string containing one or more valid Rails options, separated by spaces.
```